utils/after_success.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This script is meant to be used by Travis-CI to publish artifacts (binary, source and javadoc jars) when releasing snapshots.
+# Right now it will be manually triggered to push jars to nexus server.
+
+SITE_VERSION=$(./gradlew properties | sed -n -e 's/^\s*version: //p')
+NEXUS_SNAPSHOTS_DIR=$(cat ./build.gradle | sed -ne 's/def\s+privateSnapshotRepo\s*=\s*"\(.*\)".*/\1/p')
```

Comment:
Make a gradle task called "printSnapshotRepo" which prints the value you need. If it is a gradle task, people will know that there will be external references to it.